_projects/2023/100507931/100507931.Rmd

```diff
@@ -0,0 +1,385 @@
+---
+title: "Public trust in government near historic lows"
+description: |
+  With Republicans expressing much lower and higher levels of trust; Democrats’ attitudes have tended to be somewhat more consistent, regardless of which party controls the White House.
+categories: "2023"
+author: Ana Pérez Barrera
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false 
+    toc: true
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center")
+```
+
+The Pew Research Center has recently published an article where the evolution of public trust towards different U.S. governments between 1972 ans 2023 is shown:
+
+[INSERT IMAGE HERE]
+
+## Graph replication
+
+```{r}
+library(tidyverse)
+library(ggplot2)
+library(dplyr)
+library(tidyr)
+library(ggtext)
+```
+
+Reading and cleaning the data:
+
+```{r}
+datos <-
+  read_csv(file = "chart-export-6dcaa713-b94d-4cb5-b91e-e8d433400438 (1).csv",
+           skip = 3, quote = "\\.") |>
+  mutate(Date = mdy(str_remove_all(Date, "\t|\"")),
+         `Conservative Rep/Lean Rep` =
+           as.numeric(str_remove_all(`Conservative Rep/Lean Rep`, '"'))) |>
+  rename(pollster = .) 
+
+datos<-na.omit(datos)
+
+colnames(datos)[3] <- "Liberal_Dem"
+colnames(datos)[4] <- "Cons_Dem"
+colnames(datos)[5] <- "Liberal_Rep"
+colnames(datos)[6] <- "Cons_Rep"
+```
+
+The first step to start the replication is to create a vector of date strings for the colored area, since the background of our plot is colored by presidential terms under the main lines.
+
+```{r}
+colored_xmin <- c("1974-12-01", "1980-11-07",  "1993-01-17",  "2008-12-21",
+                  "2021-04-11")
+
+colored_xmax <- c("1977-04-25", "1989-01-15",  "2001-01-17", "2017-04-11", 
+                  "2023-06-11")
+
+# Create the intervalos data frame
+colored <- data.frame(xmin = as.Date(colored_xmin),xmax = as.Date(colored_xmax),
+                      intervalo = c("Ford",  "Reagan", "Clinton",  "Obama", "Biden" ))
+
+colores_intervalos <-  c("Ford" = "#C81518", "Reagan" = "#C81518","Clinton" = "#2E5A87",  "Obama" = "#2E5A87", "Biden" = "#2E5A87")
```

Comment:
Please restyle your code so that all lines fit into the column width. This one for instance is too long, and goes out of the page in the resulting HTML.